src/js/input.js

```diff
@@ -114,13 +117,17 @@ class Input extends Marionette.LayoutView {
     this.hintText = options.hintText || '';
     this.required = options.required || false;
     this.type = options.type || 'text';
-    this.value = options.defaultValue || '';
+    this.value = options.value || options.defaultValue || '';
   }
 
-  keyPress (event) {
-    this.value = event.target.value
-    this.onChange(this.value);
-    this.triggerMethod('input:change');
+  keyPress (e) {
+    if (e.keyCode === 13 || e.keyCode === 38 || e.keyCode === 40) {
+      e.preventDefault();
```

Comment:
so we'er focused in an input and default behavior when you press up and down is to move the cursor to beginning of the line... don't ask me why